<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CutImageDialog</class>
 <widget class="QDialog" name="CutImageDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>280</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>280</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>280</height>
   </size>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="settingGroup">
     <property name="title">
      <string>设置</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,1,0,0" rowminimumheight="24,24,24,24,24,0" columnminimumwidth="110,80,0,0">
      <property name="leftMargin">
       <number>18</number>
      </property>
      <item row="1" column="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>像素</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>切分高度</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>像素</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_1">
        <property name="text">
         <string>输出宽度</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="widthBox">
        <property name="editable">
         <bool>true</bool>
        </property>
        <item>
         <property name="text">
          <string>800</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1200</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>JPEG 质量</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="cutHeightBox">
        <property name="editable">
         <bool>true</bool>
        </property>
        <property name="currentText">
         <string>1200</string>
        </property>
        <item>
         <property name="text">
          <string>1200</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>处理模式</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QSpinBox" name="jpegQualityBox">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>80</number>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="toleranceBox">
        <property name="suffix">
         <string>%</string>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>切线色彩容差</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="modeBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <item>
         <property name="text">
          <string>自动 - 保留超过高度的图</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>自动 - 强制切分超过高度的图</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>半自动 - 手动处理超过高度的图</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>全手动</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>输出目录</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="outDirEdit"/>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="browseButton">
        <property name="text">
         <string>浏览</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>9</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>9</height>
      </size>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="startButton">
       <property name="text">
        <string>开始切分</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>outDirEdit</tabstop>
  <tabstop>browseButton</tabstop>
  <tabstop>widthBox</tabstop>
  <tabstop>cutHeightBox</tabstop>
  <tabstop>modeBox</tabstop>
  <tabstop>toleranceBox</tabstop>
  <tabstop>jpegQualityBox</tabstop>
  <tabstop>startButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
